lib/widgets/drive.coffee

```diff
@@ -51,18 +50,38 @@ getDrives = ->
 # visuals.drive('Please select a drive').then (drive) ->
 # 	console.log(drive)
 ###
-module.exports = (message = 'Select a drive') ->
-	getDrives().then (drives) ->
 
-		if _.isEmpty(drives)
-			throw new Error('No available drives')
+module.exports = (message = 'Select a drive') ->
 
-		form.ask
-			type: 'list'
-			name: 'drive'
+	getDrives().then (drives) ->
+		options =
 			message: message
+			name: 'drives'
 			choices: _.map drives, (drive) ->
 				return {
 					name: "#{drive.device} (#{drive.size}) - #{drive.description}"
 					value: drive.device
 				}
+
+		ui = new UI
+			input: process.stdin
+			output: process.stdout
+		list = new InquirerList(options, ui.rl)
+		scanner = new DriveScanner(getDrives, interval: 5000, drives: drives)
+
+		scanner.on 'add', (drive) ->
+			list.opt.choices.push
```

Comment:
Fair enough. See Inquirer's source code for the `choices` object: https://github.com/SBoudrias/Inquirer.js/blob/master/lib/objects/choices.js#L27. `realChoices` is basically `choices` without separators and disabled choices. See `push()` (https://github.com/SBoudrias/Inquirer.js/blob/master/lib/objects/choices.js#L106). It automatically adds the choice to `realChoices`.

Can you write a comment here explaining why we don't need to push to both `choices` and `readChoices`?